app/assets/stylesheets/application.css.scss

```diff
@@ -202,3 +202,16 @@ main {
   color: white;
   box-shadow: 1px 1px 10px 1px rgba(0,0,0,0.7);
 }
+
+.town_table td{
```

Comment:
Selector `town_table` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space